src/components/Country.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { FaClock } from "react-icons/fa";
+import { fetchCountries } from "../redux/countrySlice";
+import DateComponent from "../globals";
+
+const Country = () => {
+  const { countries, loading, error } = useSelector(
+    (state) => state.countrySlice
+  );
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, [dispatch]);
+  return (
+    <div>
+      {loading && <h3 className="text-center">Loading...</h3>}
+      {error && <h3>{error}</h3>}
+      <div className="grid md:grid-cols-2 mx-5 gap-10">
+        {countries.map((country) => {
+          return (
+            <div
+              className="h-max p-3 border-2 shadow-lg rounded bg-white flex flex-col justify-around"
+              key={country.country}
+            >
```

Comment:
- [ ] 🙏Please change the layout of your project to have two columns. This will make it look cool while retaining the original design.

Your design:
![image](https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/63445083/cab09da5-4fb6-435a-a546-a513642af96a)

Original design:
![image](https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/63445083/c7355914-f243-4000-adb0-ffa4fbbe8cdf)

- [ ] 🙏Please create a details page that appears when the country is clicked, as it is a project requirement. I would advise reducing some content on the home page and moving it to the details page. This will enhance the user experience and meet the project's specifications.
![image](https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/63445083/78f6d99d-5824-43de-a6e9-51282981197c)

Check below project requirement
![image](https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/63445083/ac862181-aee2-45c0-80ee-468ca8e93a99)

Reply:
this requirement is met by `Covid-19 Global Data` the very first component for hide/show funcionality